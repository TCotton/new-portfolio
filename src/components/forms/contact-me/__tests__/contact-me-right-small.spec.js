import React from 'react';
import renderer from 'react-test-renderer';
import ContactMeRightSmall from '../contact-me-right-small';

describe('ContactMeRightSmall', () => {

	let component, instance;

	beforeEach( () => {
		component = renderer.create(
			<ContactMeRightSmall />
		);
		instance = component.getInstance();
	});

	it('renders correctly', () => {
		const tree = renderer
			.create(<ContactMeRightSmall />)
			.toJSON()
		expect(tree).toMatchSnapshot();
	})

	it('should call shouldComponentUpdate', () => {
		const shouldUpdate = instance.shouldComponentUpdate();
		expect(shouldUpdate).toBe(true);
	});
})
