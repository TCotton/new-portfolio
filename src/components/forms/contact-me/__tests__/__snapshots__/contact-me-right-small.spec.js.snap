// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactMeRightSmall renders correctly 1`] = `
<div
  className="contact-right-small"
>
  <h4
    className="content-title"
  >
    Get in touch
  </h4>
  <p>
    Andy Walpole
    <br />
    Web Developer
  </p>
  <p>
    <a
      className="underline"
      href="mailto:me@andywalpole.me?Subject=Contact%20from%20portfolio"
    >
      me@andywalpole.me
    </a>
  </p>
</div>
`;
